SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -263,22 +264,41 @@ void USpatialClassInfoManager::FinishConstructingSubobjectClassInfo(const FStrin
 		check(Offset != SpatialConstants::INVALID_COMPONENT_ID);
 
 		ForAllSchemaComponentTypes([&](ESchemaComponentType Type)
-		{
-			Worker_ComponentId ComponentId = DynamicSubobjectData.SchemaComponents[Type];
-
-			if (ComponentId != SpatialConstants::INVALID_COMPONENT_ID)
 			{
-				SpecificDynamicSubobjectInfo->SchemaComponents[Type] = ComponentId;
-				ComponentToClassInfoMap.Add(ComponentId, SpecificDynamicSubobjectInfo);
-				ComponentToOffsetMap.Add(ComponentId, Offset);
-				ComponentToCategoryMap.Add(ComponentId, ESchemaComponentType(Type));
-			}
-		});
+				Worker_ComponentId ComponentId = DynamicSubobjectData.SchemaComponents[Type];
+
+				if (ComponentId != SpatialConstants::INVALID_COMPONENT_ID)
+				{
+					SpecificDynamicSubobjectInfo->SchemaComponents[Type] = ComponentId;
+					ComponentToClassInfoMap.Add(ComponentId, SpecificDynamicSubobjectInfo);
+					ComponentToOffsetMap.Add(ComponentId, Offset);
+					ComponentToCategoryMap.Add(ComponentId, ESchemaComponentType(Type));
+				}
+			});
 
 		Info->DynamicSubobjectInfo.Add(SpecificDynamicSubobjectInfo);
 	}
 }
 
+bool USpatialClassInfoManager::ShouldTrackHandoverProperties() const
```

Comment:
Perhaps too soon, but perhaps we pass the UClass here, and onto the the LBS, so we can enable/disable on a per-class basis.

Reply:
IMO it's too soon to add that surface to the API. I think this should generally be controlled by marking (or not) properties for handover in the class declarations and until we have a story for that not being sufficient I think we should leave this.